_server.js

```diff
@@ -0,0 +1,158 @@
+const electron = require('electron'),
+app = electron.app,
+BrowserWindow = electron.BrowserWindow,
+ipcMain = electron.ipcMain
```

Comment:
@usergenic `eslint` and `clang-format` are giving me conflicting advice.

This line passes the linter: `win = new BrowserWindow({width: 800, height: 600});`. When I run `clang-format`, the line is changed to: `win = new BrowserWindow({width : 800, height : 600});`, which then fails the lint test.

Which version should I keep? The formatted version since it's run after the lint version?